### PR TITLE
ACLs: add deprecation banner

### LINKED ISSUE #8056: ACLs: Update administration banner
Parent: 
https://github.com/treeverse/lakeFS/issues/8018

### PR DESCRIPTION
**Pull request by N-o-Z**
Closes #8056 

Preliminary signal for our users, expressing the intention to move ACLs

### CODE REVIEW

File: webui/src/lib/components/auth/layout.tsx [hunk 1]
```diff
@@ -28,8 +28,10 @@ export const AuthLayout = () => {
         <Container fluid="xl">
             <Row className="mt-5">
                 <div>
+                    <Alert variant="warning" title="ACL Deprecation"><InfoIcon/>{" "}<b>ACLs are moving out of core lakeFS!</b>{"  "}See our <Alert.Link href={"https://lakefs.io/blog/why-moving-acls-out-of-core-lakefs/"}>announcement</Alert.Link>{" "}
```
Comment (ozkatz) [inline webui/src/lib/components/auth/layout.tsx:31]: 60% of installations are single user anyway, and this is perhaps too intimidating. I suggest:

1. put it only in the administration tab (dunno if hard to do, but nice to have - only show if `len(users) > 1`)
2. change the wording: from "our annuncement" to "the announcement"

**Review by ozkatz — CHANGES_REQUESTED** [review]
Thanks - I agree a heads up in the product makes sense! I suggest moving it to the admin tab.

File: webui/src/lib/components/auth/layout.tsx (hunk 1 above)
Comment (N-o-Z) [reply webui/src/lib/components/auth/layout.tsx:31]: 1. It's only in the administration tab + done
2. Done

Comment (arielshaqed) [reply webui/src/lib/components/auth/layout.tsx]: Sorry to be contrary.  Consider the fact that there is little reason for a user ever to go to the Admin tab.  I would expect an admin of a medium-sized lakeFS installation to manage users maybe a few times a year.[^1]  Such an admin would never see the deprecation banner.

[^1]:  Can we estimate whether this is true?

**Review by arielshaqed — COMMENTED** [review]
Approving as code works.  However I believe that this banner will not be visible  enough to many users.

**Review by arielshaqed — APPROVED** [review]

File: webui/src/lib/components/auth/layout.tsx (hunk 1 above)
Comment (N-o-Z) [reply webui/src/lib/components/auth/layout.tsx]: Though I understand your point, I think the decision on where to show it is a product one.
@ozkatz, do you still think we should place it only on the administration tab scope?

**Review by ozkatz — APPROVED** [review]
Thanks!

File: webui/src/lib/components/auth/layout.tsx (hunk 1 above)
Comment (ozkatz) [reply webui/src/lib/components/auth/layout.tsx]: I think so. >60% of users shouldn't care about this change, no reason to instill fear. On the other hand, while I agree with @arielshaqed that the admin page isn't visited frequently, it's still probably a higher frequency than upgrading lakeFS versions. 

No perfect solution here..

**Pull request merged**